### ISSUE TITLE
go1.14 panic

### ISSUE THREAD

**Issue by xiusin** [dgraph-io/badger#1261]
### What version of Go are you using (`go version`)?

<pre>
$ go version
go version go1.14 windows/amd64
</pre>

### What version of Badger are you using?
```
github.com/dgraph-io/badger v1.6.0
```

### What are the hardware specifications of the machine (RAM, OS, Disk)?
windows10  

### What did you do?

open once, closed, next open will see panic
```
badger 2020/03/17 18:54:00 INFO: All 0 tables opened in 0s
badger 2020/03/17 18:54:00 INFO: Replaying file id: 0 at offset: 0
panic: Value log truncate required to run DB. This might result in data loss
```

### What did you see instead?
Everything is normal.

**Comment by xiusin** [comment]
sorry,

**Comment by jarifibrahim** [comment]
@xiusin FYI, badger v1.6.0 has a data loss issue on windows https://github.com/dgraph-io/badger/pull/1134 . The issue is fixed in latest release of badger v2.0.2 but v1.6.0 doesn't have this patch.

We'll soon be releasing v1.6.1 which includes https://github.com/dgraph-io/badger/pull/1134

**Comment by xiusin** [comment]
@jarifibrahim thinks
